.github/workflows/deploy.yml

```
@@ -36,15 +36,20 @@ jobs:
         RENKU_TESTS_ENABLED: true
         TEST_ARTIFACTS_PATH: "tests-artifacts-${{ github.sha }}"
         renku: "@master"
-    - name: Notify slack
-      if: success()
+    - uses: 8398a7/action-slack@v3
+      with:
+        status: custom
+        fields: job,ref
+        custom_payload: |
+          {
+            attachments: [{
+              color: '${{ job.status }}' === 'success' ? 'good' : '${{ job.status }}' === 'failure' ? 'danger' : 'warning',
+              text: `${process.env.AS_JOB} commit ${process.env.AS_COMMIT}: ${{ job.status }}.`,
+            }]
+          }
       env:
-        RENKU_SLACK_BOT_TOKEN: ${{ secrets.RENKU_SLACK_BOT_TOKEN }}
-      run: |
-        curl -X POST https://slack.com/api/chat.postMessage \
-             -H "Authorization: Bearer $RENKU_SLACK_BOT_TOKEN" \
-             --data "channel=C9U45DL1H" \
-             --data "text=https://dev.renku.ch has been updated! :tada:"
+        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      if: always()
```

```suggestion
      if: failure()
```
I'd actually switch to a notification on failure only here too. I think letting us know that everything worked as intended is pointless.

---

Currently we do notify when a new version is deployed. But I'm ok to change it if you don't think there's value in it.

---

Yes, I know. This is definitely not the main reason for our flood of notifications, but since we're touching it I'd propose to make that change now.

---

Okay, agreed 👍